contracts/wrapper/NameWrapper.sol

```diff
@@ -657,6 +644,53 @@ contract NameWrapper is
         emit NameWrapped(node, name, wrappedOwner, fuses);
     }
 
+    function _addLabelAndWrap(
+        bytes32 parentNode,
+        string memory label,
+        address newOwner,
+        uint96 _fuses
+    ) internal {
+        bytes32 labelhash = keccak256(bytes(label));
+        bytes32 node = _makeNode(parentNode, labelhash);
+        bytes memory name = _addLabel(label, names[parentNode]);
+        _wrap(node, name, newOwner, _fuses);
+    }
+
+    function _setSubnodeOwnerAndWrap(
+        bytes32 parentNode,
+        string memory label,
+        address newOwner,
+        uint96 _fuses
+    ) internal {
+        bytes32 labelhash = keccak256(bytes(label));
+        bytes32 node = _makeNode(parentNode, labelhash);
+        bytes memory name = _addLabel(label, names[parentNode]);
+        ens.setSubnodeOwner(parentNode, labelhash, address(this));
```

Comment:
This is the same as `_addLabelAndWrap` except for the call to `setSubnodeOwner`. Could that be moved into the calling function?

Reply:
Good catch, I've removed this function in favour of _addLabelAndWrap